solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/filter/SetAccessStatusFilterTest.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2017 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.data.ingest.solr.filter;
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.Mock;
+
+import edu.unc.lib.dl.acl.fcrepo4.InheritedAclFactory;
+import edu.unc.lib.dl.acl.fcrepo4.ObjectAclFactory;
+import edu.unc.lib.dl.acl.service.PatronAccess;
+import edu.unc.lib.dl.acl.util.AccessPrincipalConstants;
+import edu.unc.lib.dl.data.ingest.solr.indexing.DocumentIndexingPackage;
+import edu.unc.lib.dl.fcrepo4.ContentObject;
+import edu.unc.lib.dl.fedora.PID;
+import edu.unc.lib.dl.search.solr.model.IndexDocumentBean;
+import edu.unc.lib.dl.search.solr.util.FacetConstants;
+
+/**
+ * @author harring
+ */
+public class SetAccessStatusFilterTest {
+
+    private static final String PID_STRING = "uuid:07d9594f-310d-4095-ab67-79a1056e7430";
+
+    @Mock
+    private DocumentIndexingPackage dip;
+    @Mock
+    private IndexDocumentBean idb;
+    @Mock
+    private InheritedAclFactory inheritedAclFactory;
+    @Mock
+    private ObjectAclFactory objAclFactory;
+    @Mock
+    private ContentObject contentObj;
+    @Mock
+    private Date date;
+
+    @Mock
+    private PID pid;
+
+    @Captor
+    private ArgumentCaptor<List<String>> listCaptor;
+
+    private Map<String, Set<String>> principalRoles;
+
+    private SetAccessStatusFilter filter;
+
+    @Before
+    public void setup() throws Exception {
+        initMocks(this);
+
+        when(pid.toString()).thenReturn(PID_STRING);
+
+        when(dip.getDocument()).thenReturn(idb);
+        when(dip.getPid()).thenReturn(pid);
+        when(dip.getContentObject()).thenReturn(contentObj);
+        when(contentObj.getPid()).thenReturn(pid);
+
+        when(inheritedAclFactory.isMarkedForDeletion(any(PID.class))).thenReturn(true);
```

Comment:
its kind of weird having this happen for every test.  I'd suggest moving it down into testIsMarkedForDeletion.  It would be good to test the negative case as well to ensure that everything isn't getting marked as deleted